draco/analysis/transform.py

```diff
@@ -217,9 +224,9 @@ def pack_product_array(arr):
         # Construct the equivalent stack reverse_map for the telescope instance.  Note
         # that we identify invalid products here using an index that is the size of the stack axis.
         feedmask = pack_product_array(self.telescope.feedmask)
-        bt_rev = np.array(zip(
+        bt_rev = np.array(list(zip(
```

Comment:
Instead of an ephemeral listification, just use the iterator directly:
```suggestion
        bt_rev = np.fromiter(zip(
```

Reply:
Good catch. This is one that my syntax tree heuristics did not recognise!